podcasts/FolderViewController.swift

```diff
@@ -96,6 +98,10 @@ class FolderViewController: PCViewController, UIGestureRecognizerDelegate {
         let titleColor = ThemeColor.filterText01(filterColor: folderColor)
         let iconColor = ThemeColor.filterIcon01(filterColor: folderColor)
         let backgroundColor = ThemeColor.filterUi01(filterColor: folderColor)
+
+        if let themeableCollectionView = mainGrid as? ThemeableCollectionView {
+            themeableCollectionView.backgroundColor = backgroundColor
```

Comment:
This change causes changes such as this (Default Dark):

| Before | After |
| ------ | ----- |
| ![Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2024-04-19 at 13 17 57](https://github.com/Automattic/pocket-casts-ios/assets/7040243/0b61cc1e-b1be-4c8f-8f05-78062c117947) | ![Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2024-04-19 at 13 15 46](https://github.com/Automattic/pocket-casts-ios/assets/7040243/a7ebfc1b-54e0-45c7-9fb5-d5eba7e24875) |

Before looks better for me because the cells almost merge with the background.

Reply:
On the list view the before is the correct one, I will address this.